backend/routes/boards.js

```diff
@@ -0,0 +1,69 @@
+const { PrismaClient } = require("../generated/prisma");
+const prisma = new PrismaClient();
+
+const express = require("express");
+const { BadParams, DoesNotExist } = require("../middleware/CustomErrors");
+const router = express.Router();
+
+const app = express();
+
+router.get("/:boardId", async (req, res, next) => {
+  const boardId = parseInt(req.params.boardId);
+  const innerBoard = await prisma.board.findUnique({
+    where: {
+      id: boardId, // Replace 99 with the actual ID you want to retrieve
+    },
+  });
+  if (innerBoard) {
+    res.json(innerBoard);
+  } else {
+    next(
+      new DoesNotExist("This ID does not exist and thus cannot be displayed")
+    );
+  }
+});
+
+// create new board
+router.post("/", async (req, res, next) => {
+  const { title, category, author } = req.body;
+  if (title == null || category == null) {
+    next(new BadParams("You must include a title and category"));
+  }
+  const board = await prisma.board.create({
+    data: {
+      title,
+      category,
+      author,
+    },
+  });
+  res.status(201).json(board);
+});
+
+// delete all Boards:
```

Comment:
Looks like code below deletes just one board, not all boards, like this comment says.